Formula/libcbor.rb

```diff
@@ -13,9 +13,11 @@ class Libcbor < Formula
 
   depends_on "cmake" => :build
 
+  uses_from_macos "cjson" => :build
+
   def install
     mkdir "build" do
-      system "cmake", "-G", "Unix Makefiles", "..", *std_cmake_args
+      system "cmake", "..", *std_cmake_args
```

Comment:
This change should definitely be made.

Reply:
I think adding
```
-DCMAKE_INTERPROCEDURAL_OPTIMIZATION:BOOL=OFF
```
to the CMake args should resolve the build problems on linux. I'm not sure if the linuxbrew compiler shims are explicitly requesting LTO/IPO, but this is almost certainly the issue at hand.